dcpy/test/lifecycle/package/test_generate_data_dictionary.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from unittest import TestCase
+from dcpy.test.lifecycle.package.conftest import (
+    TEST_ASSEMBLED_PACKAGE_AND_METADATA_PATH,
+    TEST_METADATA_YAML_PATH,
+    TEMP_DATA_PATH,
+)
+
+from dcpy.lifecycle.package import generate_metadata_assets, oti_xlsx
+
+
+@pytest.mark.usefixtures("file_setup_teardown")
+class TestDataDictionary(TestCase):
+    package_path = TEST_ASSEMBLED_PACKAGE_AND_METADATA_PATH
+    yaml_file_path = TEST_METADATA_YAML_PATH
+    output_html_path = TEMP_DATA_PATH / "metadata.html"
+    output_pdf_path = TEMP_DATA_PATH / "metadata.pdf"
+    output_xlsx_path = TEMP_DATA_PATH / "my_data_dictionary.pdf"
```

Comment:
hopefully still an xlsx after this refactor?

Reply:
fixed!